cfg/rgl_base.cfg

```diff
@@ -126,7 +124,6 @@ tv_transmitall "1"                               // transmits all entity data fr
                                                  // ^ THIS DOES NOT REPLACE SOURCETV+, which can be found here: https://github.com/dalegaard/srctvplus
 tv_enable "1"                                    // enables Source TV
 
-sv_turbophysics "1"                              // prevents people from moving physics objects by shooting them or running into them when combined with disabling +use. results in less CPU usage on the server
```

Comment:
comment is for if turbophysics is off, not on, also you don't want this off, it will break passtime maps and other maps with phys for no realistic performance gain

Reply:
i don't really understand why you're removing this line entirely? you don't want it off, so the line should stay, correct?
additionally, the [valve dev wiki](https://developer.valvesoftware.com/wiki/Sv_turbophysics) states that `When sv_turbophysics is non-zero, players can't move physics objects by touch. To move them in turbophysics mode, they must press the use key.` i've tested it and force from shooting the passtime ball with hitscan or projectiles does not change based on the value of `sv_turbophysics`. maybe this affects shooting other physics objects tho?

Reply:
it's not shooting the ball, it was some strange issue where occasionally (not all the time) the passtime ball would be weird when you picked it up. i don't remember why i deleted it, i don't think there is a reason to delete it?